tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/DdwafResultStruct.cs

```diff
@@ -11,13 +11,12 @@ namespace Datadog.Trace.AppSec.Waf.NativeBindings
     [StructLayout(LayoutKind.Sequential)]
     internal struct DdwafResultStruct
     {
-        [Obsolete("This member will be removed from then ddwaf library by a future PR")]
-        public DDWAF_RET_CODE Action;
+        public bool Timeout;
+
+        public int PerfTotalRuntime;
 
         public IntPtr Data;
```

Comment:
Same question, do we have potential version-conflict crashing scenarios here?

Reply:
With an older version return an action code, it still works locally for me in that case, but same as my previous answer, it's a really good point and in the future, we should be careful to potential breaking changes from upcoming waf versions.